src/containers/HeaderContainer/HeaderContainer.vue

```diff
@@ -89,6 +89,24 @@
                 <b-nav-item v-if="isHomePage" to="/#about-mew">{{
                   $t('header.about')
                 }}</b-nav-item>
+                <b-nav-item-dropdown
+                  v-if="wallet !== null"
+                  right
+                  no-caret
+                  class="tx-history-menu"
+                >
+                  <template slot="button-content">
+                    <p>Transaction History</p>
+                  </template>
+                  <b-dropdown-item :href="blockExplorerURL" target="_blank">
+                    <p>Etherscan (ETH)</p></b-dropdown-item
```

Comment:
it is not always Etherscan, we have to switch based on the service provider

Reply:
You can probably move the `getService` function on SuccessModal to helpers so you can use it here too.